src/state_point.F90

```diff
@@ -170,8 +181,7 @@ subroutine write_state_point()
           mesh_group = create_group(meshes_group, "mesh " // trim(to_str(meshp%id)))
 
           call write_dataset(mesh_group, "id", meshp%id)
-          call write_dataset(mesh_group, "type", meshp%type)
-          call write_dataset(mesh_group, "n_dimension", meshp%n_dimension)
+          call write_dataset(mesh_group, "type", "regular")
```

Comment:
Why not keep as the more general ``mesh%type`` assuming we will one day have hex and/or other structured tally mesh types?